### ISSUE TITLE
E363: pattern uses more memory than 'maxmempattern'

### ISSUE THREAD

**Issue by Garzet** [aklt/plantuml-syntax#82]
I think there is a bug with highlighting. Error E363 happens when 35 of more dashes ('-') are in continuation. I use those in comments when I want to delimit sections of the document.

I don't know enough about syntax files, but I guess it should be an easy fix for someone who understands them.

**Comment by aklt** [comment]
I could not reproduce this.  Is it still a problem?

**Comment by Garzet** [comment]
Yep, I still have the issue. I have switched to neovim in the meantime (version 0.4.3).
Opening file `test.puml` with this contents results with E363: `-----------------------------------`

**Comment by aklt** [comment]
Thanks for letting me know about this one, I will see if I can fix it.
